src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -987,7 +987,7 @@ protected void CheckFeatureAvailabilityAndRuntimeSupport(SyntaxNode declarationS
                     diagnostics.Add(ErrorCode.ERR_RuntimeDoesNotSupportDefaultInterfaceImplementation, location);
                 }
 
-                if (((!hasBody && IsAbstract) || IsVirtual) && IsStatic && !ContainingAssembly.RuntimeSupportsStaticAbstractMembersInInterfaces)
+                if (((!hasBody && IsAbstract) || IsVirtual) && !isExplicitInterfaceImplementation && IsStatic && !ContainingAssembly.RuntimeSupportsStaticAbstractMembersInInterfaces)
```

Comment:
>isExplicitInterfaceImplementation [](http://example.com/codeflow?start=64&length=33)

Do we need the additional parameter given that we have `IsExplicitInterfaceImplementation` property (used in this method already)?

Reply:
> Do we need the additional parameter given that we have ```IsExplicitInterfaceImplementation``` property (used in this method already)?

Good catch, I will follow up on this in a separate PR.